### PR TITLE
Add config test to mergelists

### PR DESCRIPTION
**Pull request by chases2**
Updates TestGrid version as well.
The version update caused unit testing to enforce requirements on local default files. This may fix https://github.com/kubernetes/test-infra/issues/20756

/assign @fgimenez 
I'm looking for feedback on the default file. Now that https://github.com/kubernetes/test-infra/pull/20902 is in, should the default file be updated (as shown) or removed entirely?

/assign @mpherman2 
The strict default config option was set to "false" for now.

Follow-up to https://github.com/kubernetes/test-infra/pull/20986

/lint

### CODE REVIEW

**Comment by fgimenez** [comment]
> I'm looking for feedback on the default file. Now that #20902 is in, should the default file be updated (as shown) or removed entirely?

@chases2 thanks for the heads up! I think it is ok to completely remove `config/testgrids/kubevirt/default.yaml`, we only needed the `open_test_template.url` customization and is now working fine with #20902

**Comment by chases2** [comment]
Deleted the unused default.yaml

Update-deps updates TestGrid and changes the go.mod and go.sum files, but it does so in a way that verify-deps won't accept as accurate.

**Comment by chases2** [comment]
Found the root issue: https://github.com/kubernetes/test-infra/issues/21463

This PR should be good to go now

File: prow/spyglass/lenses/junit/lens.go [hunk 1]
```diff
@@ -158,7 +158,7 @@ func (lens Lens) getJvd(artifacts []api.Artifact) JVD {
 				resultChan <- result
 				return
 			}
-			var suites junit.Suites
+			var suites *junit.Suites
```
Comment (spiffxp) [inline prow/spyglass/lenses/junit/lens.go:161]: ???

this seems unrelated
